geoviews/util.py

```diff
@@ -652,9 +654,14 @@ def from_xarray(da, crs=None, apply_transform=False, nan_nodata=False, **kwargs)
                   'defaulting to non-geographic element.')
     elif hasattr(da, 'rio') and da.rio.crs is not None:
         # rioxarray.open_rasterio
-        try:
-            kwargs['crs'] = process_crs(da.rio.crs.to_proj4())
-        except Exception:
+        crs = None
+        for n in ("to_epsg", "to_proj4"):
+            with suppress(Exception):
+                crs = process_crs(getattr(da.rio.crs, n)())
```

Comment:
Why is it called n?

Maybe call in method?
```suggestion
        for method_name in ("to_epsg", "to_proj4"):
            with suppress(Exception):
                crs = process_crs(getattr(da.rio.crs, method_name)())
```